pkgs/applications/networking/instant-messengers/signal-desktop/default.nix

```diff
@@ -1,8 +1,8 @@
 { callPackage }: builtins.mapAttrs (pname: attrs: callPackage ./generic.nix (attrs // { inherit pname; })) {
   signal-desktop = {
     dir = "Signal";
-    version = "6.0.1";
-    hash = "sha256-7Cojhz3wBPd/13uVg2MgJXvR9QMPZcwBibk/sCrRMAE=";
+    version = "6.1.0";
+    hash = "sha256-70IQ/2yjHbez8SpZxqZKa/XWIEYA3cN7JAIM9kgjN30=";
   };
   signal-desktop-beta = {
```

Comment:
can we get in the habit of updating `signal-desktop-beta` at the same time? they have shipped `6.2.0-beta.1`

Reply:
That's in #206313